src/remoteConnector.ts

```diff
@@ -1030,11 +1031,12 @@ export default class RemoteConnector extends Disposable {
 
 		await this.context.globalState.update(`${RemoteConnector.SSH_DEST_KEY}${sshDestStr}`, { ...connectionInfo, isFirstConnection: false });
 
+		const heartbeatSupported = session.scopes.includes(ScopeFeature.LocalHeartbeat);
 		const gitpodVersion = await getGitpodVersion(connectionInfo.gitpodHost, this.logger);
-		if (isFeatureSupported(gitpodVersion, 'localHeartbeat')) {
+		if (heartbeatSupported) {
 			this.startHeartBeat(session.accessToken, connectionInfo, gitpodVersion);
 		} else {
-			this.logger.warn(`Local heatbeat not supported in ${connectionInfo.gitpodHost}, using version ${gitpodVersion.version}`);
+			this.logger.warn(`Local heatbeat not supported in ${connectionInfo.gitpodHost}, using version ${gitpodVersion.raw}`);
```

Comment:
(suggestion, minor): typo
```suggestion
			this.logger.warn(`Local heartbeat not supported in ${connectionInfo.gitpodHost}, using version ${gitpodVersion.raw}`);
```